config/packages/doctrine_migrations.yaml

```diff
@@ -1,5 +1,4 @@
 doctrine_migrations:
-    dir_name: "%kernel.project_dir%/src/Migrations"
-
-    # Namespace is arbitrary but should be different from App\Migrations as migrations classes should NOT be autoloaded
-    namespace: DoctrineMigrations
+    storage:
```

Comment:
Shouldn't you configurate here `migrations_path` for migration from Demo?

Reply:
I think that it is no longer needed https://github.com/Sylius/SyliusDemo/pull/150#discussion_r478370085

Reply:
The comment applies to configuration from `test_cached` that is not needed, because it is the same as the main configuration. It could be not needed, but to be honest, I'm not sure about that